auth/auth/auth.py

```diff
@@ -558,6 +559,14 @@ async def userinfo(request):
     return web.json_response(user)
 
 
+@routes.get('/api/v1alpha/version')
+async def rest_get_version(request):  # pylint: disable=W0613
```

Comment:
Why do you want to know the version of hailtop used by the auth service? I thought you wanted to add this to the query service.

Reply:
Thanks @danking, I had initially thought the two were linked, and auth might just be a better place for it. But I see that putting this on the query service is much closer to what we want. I've moved the version endpoint to the query endpoint and tested this works on our local instance.

Thanks!